src/Aquifer.API/Endpoints/Help/Endpoint.cs

```diff
@@ -0,0 +1,43 @@
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.API.Endpoints.Help;
+
+public class Endpoint(AquiferDbContext dbContext) : EndpointWithoutRequest<Response>
+{
+    public override void Configure()
+    {
+        Get("/help");
+    }
+
+    public override async Task HandleAsync(CancellationToken ct)
+    {
+        var response = new Response
+        {
+            Releases = await GetReleasesAsync(ct),
+            HowTos = await GetHowTosAsync(ct)
+        };
+        await SendOkAsync(response, ct);
+    }
+
+    private async Task<List<HelpDocumentEntity>> GetReleasesAsync(CancellationToken ct)
+    {
+        return await dbContext
+            .HelpDocuments
+            .Where(x => x.Type == HelpDocumentType.Release)
+            .OrderByDescending(x => x.Updated)
+            .Take(10)
```

Comment:
Where did this requirement to only grab the last 10 come from? I'm not seeing that in the ticket.